<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVEditor</class>
 <widget class="QMainWindow" name="CSVEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="grid">
          <item>
           <widget class="QPushButton" name="pushAddRow">
            <property name="toolTip">
             <string>Insert row</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/add_row.png</normaloff>icons/add_row.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushDeleteRow">
            <property name="toolTip">
             <string>Remove row</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/delete_row.png</normaloff>icons/delete_row.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushAddColumn">
            <property name="toolTip">
             <string>Add column</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/add_column.png</normaloff>icons/add_column.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushDeleteColumn">
            <property name="toolTip">
             <string>Remove column</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/delete_column.png</normaloff>icons/delete_column.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushSwapXY">
            <property name="toolTip">
             <string>Swap X/Y</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/swap.png</normaloff>icons/swap.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="csvInfoLabel">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>170</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>170</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="2">
             <widget class="QComboBox" name="columnDivComboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>\t</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>,</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>\n</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Seperator:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Row-seperator:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="rowDivComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>\n</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>\t</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>,</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <attribute name="verticalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Profiles: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="profileComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>37</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadProfileButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/open_profile.png</normaloff>icons/open_profile.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addProfileButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/add.png</normaloff>icons/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeProfileButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/remove.png</normaloff>icons/remove.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Signals</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="signalListWidget">
          <item>
           <property name="text">
            <string>+</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="text">
           <string>Import signals</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1363</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/menu.png</normaloff>icons/menu.png</iconset>
    </property>
    <addaction name="pushButtonLoad"/>
    <addaction name="pushButtonWrite"/>
    <addaction name="separator"/>
    <addaction name="pushButtonPreview"/>
    <addaction name="pushButtonPrint"/>
    <addaction name="separator"/>
    <addaction name="pushClear"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <action name="pushButtonLoad">
   <property name="text">
    <string>Open CSV</string>
   </property>
  </action>
  <action name="pushClear">
   <property name="text">
    <string>Close CSV file</string>
   </property>
  </action>
  <action name="pushButtonWrite">
   <property name="text">
    <string>Save as CSV</string>
   </property>
  </action>
  <action name="pushButtonPreview">
   <property name="text">
    <string>Print-Preview</string>
   </property>
  </action>
  <action name="pushButtonPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushAddRow</tabstop>
  <tabstop>pushDeleteRow</tabstop>
  <tabstop>pushAddColumn</tabstop>
  <tabstop>pushDeleteColumn</tabstop>
  <tabstop>pushSwapXY</tabstop>
  <tabstop>columnDivComboBox</tabstop>
  <tabstop>rowDivComboBox</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>profileComboBox</tabstop>
  <tabstop>loadProfileButton</tabstop>
  <tabstop>addProfileButton</tabstop>
  <tabstop>removeProfileButton</tabstop>
  <tabstop>signalListWidget</tabstop>
  <tabstop>importButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
